README.md

```diff
@@ -107,3 +108,25 @@ docker compose up -d
 ```shell
 npm run start:dev
 ```
+
+### Create jwt for dev usage
+
+By default, all calls to the `node-template` app will require a valid JWT token.
+
+You have multiple options to ease your development:
+1. Comment `onRequest` hook for JWT verification [here](./src/plugins/jwtTokenPlugin.ts)
+2. Generate a valid JWT with provided [generateJwt](./scripts/generateJwt.ts) script
+
+#### Generate JWT script
+
+- Script requires public-private key pair encrypted with `RS256` algorithm.
+- Put generated keys [here](./scripts/keys), default file names are `jwtRS256.key` for private key and `jwtRS256.key.pub` 
```

Comment:
linking some kind of instruction for generating keys may be helpful

Reply:
fixed